packages/quantic/force-app/main/default/lwc/quanticResultTemplate/quanticResultTemplate.js

```diff
@@ -19,6 +19,18 @@ import {LightningElement} from 'lwc';
  * </c-quantic-result-template>
  */
 export default class QuanticResultTemplate extends LightningElement {
+  /**
+   * @api
+   * @type {boolean}
+   */
+  @api isAnyPreviewOpen = false;
```

Comment:
Have you tested how this is interpreted by the automatic doc generator? I think it takes into account any attribute with the @api tag in the jsdoc. Given this is an internal prop we don't want users to modify, it should not be picked up by the reference doc. I think removing @api on L23 would do it.

Reply:
you are right 👌 the `@api` need to be removed here!